mbedtls-sys/build/bindgen.rs

```diff
@@ -37,37 +86,31 @@ impl super::BuildConfig {
 
         let include = self.mbedtls_src.join("include");
 
-        let logger = StderrLogger;
-        let mut bindgen = bindgen::Builder::new(header.into_os_string().into_string().unwrap());
-        let bindings = bindgen
-            .log(&logger)
+        let bindings = bindgen::builder()
             .clang_arg("-Dmbedtls_t_udbl=mbedtls_t_udbl;") // bindgen can't handle unused uint128
             .clang_arg(format!(
                 "-DMBEDTLS_CONFIG_FILE=<{}>",
                 self.config_h.to_str().expect("config.h UTF-8 error")
             )).clang_arg(format!(
                 "-I{}",
                 include.to_str().expect("include/ UTF-8 error")
-            )).match_pat(include.to_str().expect("include/ UTF-8 error"))
-            .match_pat(self.config_h.to_str().expect("config.h UTF-8 error"))
-            .use_core(true)
+            )).header(
+                header
+                    .to_str()
+                    .expect("failed to convert header path to string"),
+            ).use_core()
             .derive_debug(false) // buggy :(
```

Comment:
I will investigate if u128 and derive_debug are now working. Do you remember what the issue was with derive_debug?

Reply:
No, it's been a long time. Possibly it would add `#[derive(Debug)]` on types where not all fields where themselves `Debug`, resulting in compilation failure.